official/benchmark/keras_imagenet_benchmark.py

```diff
@@ -995,7 +996,7 @@ def benchmark_8_gpu_fp16_tweaked(self):
     FLAGS.model_dir = self._get_model_dir('benchmark_8_gpu_fp16_tweaked')
     FLAGS.batch_size = 256 * 8  # 8 GPUs
     FLAGS.tf_gpu_thread_mode = 'gpu_private'
-    FLAGS.dataset_num_private_threads = 40
+    FLAGS.datasets_num_private_threads = 40
```

Comment:
Still not reverted.

Reply:
@alanchiao
Currently if you run the benchmark tests in `MobilenetV1KerasClusteringBenchmarkReal`, when it hits `benchmark_8_gpu_fp16_tweaked` and `benchmark_8_gpu_fp16_dynamic_tweaked`, it will throw `NotImplementedError: Clustering is currently only supported on dtype=tf.float32`. However, before I implemented this `NotImplementedError`, it throws an error `absl.flags._exceptions.UnrecognizedFlagError: Unknown command line flag 'dataset_num_private_threads'` because the flag `dataset_num_private_threads` is wrong I believe. However, I am aware of the fact that it is out of the scope of this PR. If you want this change to be reverted and deal with it later, I think it is also good.

Reply:
Looks like you're right - I'm assuming this is a bug that wasn't caught and isn't regularly run. Good to leave as is then.

@saberkun  for visibility.